src/pykx/pandas_api/pandas_meta.py

```diff
@@ -228,6 +228,21 @@ def max(self, axis=0, skipna=True, numeric_only=False):
             res
         ), cols)
 
+    @convert_result
+    def idxmax(self, axis=0, skipna=True, numeric_only=False):
+        res, cols = preparse_computations(self, axis, skipna, numeric_only)
+        maximums = self.max(axis, skipna, numeric_only)
```

Comment:
Minor: instead of using `self` around the `pandas_meta.py` file they do a `tab = self` first and then proceed with the logic. Try to adhere to the library standards